R/write.R

```diff
@@ -216,9 +220,16 @@ write_data2 <- function(
     na.strings = na_strings(),
     data_table = FALSE,
     inline_strings = TRUE,
-    dims = NULL
+    dims = NULL,
```

Comment:
Maybe adding dims to `write_xlsx()` could be its own PR if there are too manu moving parts. Would need to check `rows`, `cols`, `start_row` etc. ?

Reply:
Interesting, I wasn't aware that we do not support `dims` in `write_xlsx()`, but maybe that's not so important. Presumably people that use `write_xlsx()` only want a data frame or vector written as xlsx file and do not care that much about styling.